Inventory adjustments (#38)
msupply-foundation/msupply_docs: content/docs/inventory/stock_view.md

Barcodes updated in this way will also synchronised with other stores, which means that codes you scan here will allow other stores to automatically add these items to Outbound Shipments using a barcode scanner.

### Log tab

To see changes made to this stock line, you can click the `Log` tab. The list shows details of each change, along with the date and time of the change and the user that made the change.

<div class="imagetitle" style="margin-bottom: 40px;">Log, showing changes made to this batch</div>

![Stock line log](/docs/inventory/images/stock_line_edit_log.png)

### Ledger tab

> The `Log` tab shows changes relating to batch information, such as changes in location or pricing. The `Ledger` tab shows the stock movements for a particular stock line. These could be the result of Inbound/Outbound Shipments, Returns, Repacks or Inventory Adjustments.

Reviewer: ```suggestion
The `Log` tab shows changes relating to the batch, such as changes in location or pricing. The `Ledger` tab shows the stock movements for a particular stock line. These could be the result of Inbound/Outbound Shipments, Returns, Repacks or Inventory Adjustments.
```

![Repack entry form](/docs/inventory/images/repack-enter.png)

> From here, you can enter the number of packs which you'd like to repack, up to a maximum of the current number of packs in stock. This number is the number shown next to `Packs available` - in this case there are 5779 packs available.

Reviewer: ```suggestion
From here, you can enter the number of packs which you'd like to repack, up to a maximum of the current number of packs in stock (represented by the number next to `Packs available`) - in this case there are 5779 packs available.
```
